modules/enable-openshift-monitoring-central-helm.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * configuration/monitor-acs.adoc
+:_module-type: PROCEDURE
+[id="enable-monitoring-central-helm_{context}"]
+= Monitoring Central services using Helm charts
+
+[role="_abstract"]
+You can monitor Central services, Central and Scanner, by changing the configuration options in the `central-services` Helm chart.
+
+.Procedure
+. Update the `values-public.yaml` configuration file with the following values:
++
+[source,yaml]
+----
+monitoring.openshift.enabled: true
+----
+. Run the `helm upgrade` command and specify the configuration files using the `-f` option:
++
+[source,terminal]
+----
+$ helm upgrade -n stackrox \
+  stackrox-central-services rhacs/central-services \
+  -f <path_to_values_public.yaml>
+----
++
+[NOTE]
+====
+You can also specify configuration values using the `--set` or `--set-file` parameters. However, these options are not saved, and it requires you to manually specify all the options again whenever you make changes.
```

Comment:
```suggestion
You can also specify configuration values using the `--set` or `--set-file` parameters. However, these options are not saved, and you must manually specify all of the options again whenever you make changes.
```

Reply:
Helm does save the options, and one can use them e.g. via `--reuse-values`. In any case, I don't think we should be too specific about how Helm works.